Sources/libhostmgr/CLI/ExitCode.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+public enum ExitCode: Int32, Error {
+    case fileNotFound
+    case unableToFindRemoteImage
+    case unableToImportVM
+    case invalidVMStatus
+    case notEnoughLocalDiskSpace
+    case parallelsVirtualMachineDoesNotExist
+    case parallelsVirtualMachineIsNotStopped
+    case parallelsVirtualMachineAlreadyExists
+}
```

Comment:
@jkmassel Doesn't that mean that `ExitCode.fileNotFound` has an `Int32` value of `0`, which is reserved for success? So basically, `Console().crash("Woops", .fileNotFound)` will `exit 0` and make things appear as if all went OK…